src/State/Hub.php

```diff
@@ -278,12 +279,17 @@ public function startTransaction(TransactionContext $context, array $customSampl
 
                 $sampleSource = 'config:traces_sampler';
             } else {
-                $sampleRate = $this->getSampleRate(
-                    $samplingContext->getParentSampled(),
-                    $options->getTracesSampleRate() ?? 0
-                );
-
-                $sampleSource = $samplingContext->getParentSampled() ? 'parent' : 'config:traces_sample_rate';
+                $parentSampleRate = $context->getMetadata()->getParentSamplingRate();
+                if ($parentSampleRate !== null) {
+                    $sampleRate = $parentSampleRate;
+                    $sampleSource = 'parent';
```

Comment:
Should we make this distinct to indicate we take the parent sample rate instead of the parent sampling decision?

Reply:
Updated, wdyt?

Reply:
🪨